livekit_internal.proto

```diff
@@ -150,6 +150,7 @@ message StartSession {
   //if reconnect, client will set current sid
   string participant_id = 16;
   ReconnectReason reconnect_reason = 17;
+  bool subscriber_allow_pause = 18;
```

Comment:
just as a question: is this a publisher side or a subscriber side setting? i.e. does this indicate `tracks published by this user are allowed to be paused on subscribers` or `allow all the tracks that this user subscribed to, to be paused`

Reply:
the latter @lukasIO . This is a participant telling that my down tracks are allowed to pause or not. Please let me know if you can think of a better name.